solutions/swb-reference/README.md

```diff
@@ -6,348 +6,736 @@ The 0.1.0 beta release is available for customers to test the newly re-architect
-3. Uncomment the `stage` attribute and provide the correct `<STAGE>` value for the attribute
-4. Uncomment `awsRegion` and `awsRegionShortName`. `aws-region` value can be one of the values on this [table](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html#Concepts.RegionsAndAvailabilityZones.Regions), under the `Region` column. `awsRegionName` can be a two or three letter abbreviation for that region, of your own choosing.
-5. Uncomment `rootUserEmail` and provide the main account user's email address
-6. Run `chmod 777 <STAGE>.yaml` to allow local script to read the file
+Rush Multi-Project Build Tool 5.62.1 - Node.js 14.17.0 (LTS)
+> "cdk bootstrap"
 
-Note: For Solutions Implementations deployment, the `awsRegionShortName` value will be pre-assigned to `test`.
+ ⏳  Bootstrapping environment aws://123456789012/ap-northeast-3...
+Trusted accounts for deployment: (none)
+Trusted accounts for lookup: (none)
+Using default execution policy of 'arn:aws:iam::aws:policy/AdministratorAccess'. Pass '--cloudformation-execution-policies' to customize.
+CDKToolkit: creating CloudFormation changeset...
+ ✅  Environment aws://123456789012/ap-northeast-3 bootstrapped.
+```
 
-## Running Code Locally
+### Deploy the code
+In `swb-reference` root directory run the following code
 
-If you have made changes to the `environment` package or the `swb-reference` package follow these steps
+```bash
+STAGE=<STAGE> rushx cdk-deploy              # Deploy code to `Main Account` on AWS
+```
+Take note of the following Cloudformation outputs. We will be using them in future steps.
+```
+S3BucketArtifactsArnOutput
+AccountHandlerLambdaRoleOutput
+ApiLambdaRoleOutput
+StatusHandlerLambdaRoleOutput
+APIGatewayAPIEndpoint
+DataSetsBucketName
+```
 
-1. In `solution-spark-on-aws` root directory run `rush build`
-2. In `solution-spark-on-aws/solutions/swb-reference` root directory run `STAGE=<STAGE TO RUN LOCALLY> ./scripts/runLocally.sh`. This will run a local lambda server.
+Run the post deployment step
+```
+STAGE=<STAGE> rushx run-postDeployment      # Setup Service Catalog portfolio and products
+```
+After post deployment is complete a temporary password will be sent to the main account user's email, take note of the temporary password as its going to be needed for API authenticated requests.
+
+## Deploy to the Hosting Account
+After the deployment succeeds, we will need to set up the `Hosting account`
+1. Log into your AWS `Hosting Account` and go to Cloudformation
+1. Choose to create a new stack. On the prompt `Create Stack`, choose `Upload a template file`. 
+1. Upload the corresponding .yaml file from [templates](./src/templates) depending on your installation type. Default installation uses [onboard-account.cfn.yaml](./src/templates/onboard-account.cfn.yaml), for BYON or TGW see [Hosting Account Templates](./src/templates/README.md)
```

Comment:
Instead of "for BYON or TGW see", write "for more customizable network options, refer to"

Reply:
added new text